src/main/scala/sbtjflex/SbtJFlexPlugin.scala

```diff
@@ -43,46 +42,42 @@ object SbtJFlexPlugin extends AutoPlugin {
     * use this if you don't want jflex to run automatically (because, e.g., you're checking it in)
     * you'll want to set [[target]] in [[jflex]] using [[unmanagedJflexSettings]] or your own variant
     */
-  lazy val commonJflexSettings: Seq[Def.Setting[_]] = inConfig(jflex)(Seq(
+  lazy val commonJflexSettings: Seq[Def.Setting[_]] = inConfig(Jflex)(Seq(
     toolConfiguration := JFlexToolConfiguration(),
     pluginConfiguration := PluginConfiguration(),
     jflexDependency := "de.jflex" % "jflex" % "1.6.1",
 
-    sourceDirectory <<= (sourceDirectory in Compile) { _ / "flex" },
+    sourceDirectory := (sourceDirectory in Compile).value / "flex",
 
-    managedClasspath <<= (classpathTypes in jflex, update) map { (ct, report) =>
-      Classpaths.managedJars(jflex, ct, report)
-    },
+    managedClasspath := Classpaths.managedJars(Jflex, (classpathTypes in Jflex).value, update.value),
 
-    generate <<= sourceGeneratorTask
+    generate := {
+      val out = streams.value
```

Comment:
i thought you really aren't supposed to use `.value` because it messes up SBT's dependency tracking? is that ok now?

Reply:
It's the only option now so...I hope so.

Reply:
I couldn't find any reports about problems with the `.value` macro, but I've also played around with it a bit and it seems not to cause problems to me. If you have any reports handy I can test for specific cases of course.

Reply:
oh right, they made it special now